test/is.test.js

```diff
@@ -108,24 +108,44 @@ describe('`is`', function () {
             checkThat: 'https://youtube.come/qwerty',
             is: false
         },
-        'checks valid steam id': {
+        'checks valid steam profile name': {
+            fn: 'steam.name',
+            checkThat: 'abcdefgh123',
+            is: true
+        },
+        'checks invalid steam profile name': {
+            fn: 'steam.name',
+            checkThat: ' ',
+            is: false
+        },
+        'checks valid steam profile id': {
             fn: 'steam.id',
-            checkThat: '12345678901234567',
+            checkThat: '1234567890123456',
             is: true
         },
-        'checks invalid steam id': {
+        'checks invalid steam profile id': {
             fn: 'steam.id',
-            checkThat: 'qwert.y',
+            checkThat: 'x',
+            is: false
+        },
+        'checks valid steam profile id': {
+            fn: 'steam.profileUrl',
+            checkThat: 'https://steamcommunity.com/profiles/1234567890123456',
```

Comment:
Ah ok, in that case, could we allow for the length to vary at https://github.com/WatchBeam/is-social/blob/9b8e6e1fe0d12def59f93f93c37c14bfb6a6685f/lib/is.js#L79 rather than require exactly 16 characters?